local-interchain/interchain/handlers/actions.go

```diff
@@ -216,21 +218,49 @@ func (a *actions) relayerCheck(w http.ResponseWriter, r *http.Request) error {
 	return err
 }
 
-func KillAll(ctx context.Context, ic *interchaintest.Interchain, vals map[string][]*cosmos.ChainNode, relayer ibc.Relayer, eRep ibc.RelayerExecReporter) {
-	if relayer != nil {
-		if err := relayer.StopRelayer(ctx, eRep); err != nil {
+// KillAllICTContainers kills all containers that are prefixed with interchaintest specific container names.
+// This is required as you can not ctrl+c kill a local-ic instance before the ic.Build(), else containers will remain running.
+// (Since the vals map does not have context to these containers yet - due to still being built)
+func KillAllICTContainers(ctx context.Context) {
```

Comment:
This could be a useful interchaintest cleanup method. May move to ict vs being in local-ic. Thoughts?

Reply:
Does the current method not successfully close all of the ICT containers? What benefits would this have outside of local-ic

Reply:
It does not during ic.Build(...), if it fails within the containers persist. This would allow for someone to run into that, and then clean up.

Though keeping the containers can be useful for debugging, so we would not force this on ICT users, just allow them to use it if they see the need (like I did).

Reply:
I moved to dockerutil since that is a better place for it anyways. Just added returning a []string of the instances removed rather than printing out (so its up to the app dev)

d10ae057221274a222e2b634258b5c5aea30db20